prac_09/sort_files_2.py

```diff
@@ -0,0 +1,47 @@
+""" Processes the files in the directory and sorts them into their appropriate folders. """
+import shutil
+import os
+
+
+def main():
+    """ Sort through the files in the directory and put them into their appropriate extension directories """
```

Comment:
I would remove the spaces at the start and end of the docstrings and add a full stop.
```suggestion
    """Sort through the files in the directory and put them into their appropriate extension directories."""
```